packages/oscal-react-library/src/components/oscal-utils/OSCALCatalogUtils.ts

```diff
@@ -7,10 +7,10 @@ import matchingProp from "./OSCALPropUtils";
  * @param control A catalog control
  * @returns true if control is withdrawn
  */
-export default function isWithdrawn(control: Control): boolean {
+export default function isWithdrawn(item: Control | ControlGroup): boolean {
```

Comment:
```suggestion
interface Propable {
  props?: Property[];
}
export default function isWithdrawn(item: Propable): boolean {
```

Not sure if this is good or dumb but it's an idea I had while implementing this. I'd have to read and see what besides these two could be withdrawn.